OKTA-459416 Custom Self-Service Pwd Recovery Guide for Java (#3292)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/aspnet/emailtemplate.md

> If you're using the Sign-In Widget sample app or the SDK sample app use `https://localhost:44314/magiclink/callback?otp=${oneTimePassword}&state=${request.relayState}`. The following shows the resulting `<a>` element in the email template:

Reviewer: Can we simply replace "If you're using" with "For"... it conveys much the same information in a fewer words and doesn't require as much cognitive load (When used outside of a clause that's introduced the idea of conditional cases, "If you're" sets the person up to look for some unusual condition... it's subtle, but it makes a more pleasant experience for the reader to simply read "For...")

```suggestion
For the Sign-In Widget or SDK sample apps use `https://localhost:44314/magiclink/callback?otp=${oneTimePassword}&state=${request.relayState}`. This shows the resulting `<a>` element in the email template:
```

Author: @mauricesharp-okta  This whole section has been completely rewritten. It now reads

Replace the ${resetPasswordLink} variable with the URL for the endpoint in your application that processes the magic link. You must append the ${oneTimePassword} and ${request.relayState} variables as query parameter values. For example, if you're using one of the sample apps, the updated link is as follows:

<a id="reset-password-link"
   href="http://localhost:8080/magic-link/callback?otp=${oneTimePassword}&state=${request.relayState}"
   style="text-decoration: none;">
   <span style="padding: 9px ...;">
       Reset Password
   </span>
</a>
